pkg/istioversions/versions.yaml

```
@@ -10,6 +10,15 @@
 # updated to match the most recent version specified here. The versions in
 # go.mod affect the generated API schema for the Sail CRDs (e.g. IstioRevision),
 # as well as all the Istio CRDs (e.g. VirtualService).
+alias:
```

Use plural for consistency with `versions`
```suggestion
aliases:
```

But why is this a separate list? Why don't we add aliases to the `versions` list? By having two lists, we lose the ability to specify all the versions in a specific order (not sure if we ever need this, but we might).

---

Updated the naming. I chose this structure because I was thinking we could split the alias definitions into a separate file.

---

I don't see why we'd want this to be in a separate file, but I do see why we'd want to have a single list of versions, so that we can have each alias in the correct place rather than having all of the aliases at the end.